### PR TITLE
storage: writeMemorySeries needs to return true for quarantined series

### PR DESCRIPTION
**Pull request by beorn7**
This is another fallout of my bug hunt.

@brian-brazil Sorry for missing this one previously. I found it with the help of my added panics on invariants. It's a very rare occurrence, luckily.

### CODE REVIEW

**Comment by beorn7** [comment]
Fixes #2409

File: storage/local/storage.go [hunk 1]
```diff
@@ -1474,7 +1479,7 @@ func (s *MemorySeriesStorage) writeMemorySeries(
 		var offset int
 		offset, persistErr = s.persistence.persistChunks(fp, chunks)
```
Comment (brian-brazil) [inline storage/local/storage.go]: persistChunks's comment says it's the responsibility of the caller to quarantine the series, similarly with the others looking at their code.

Comment (beorn7) [inline storage/local/storage.go]: Yes, that's happening in L1452.

Comment (brian-brazil) [inline storage/local/storage.go]: Ah, missed that.

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**